code/modules/jobs/job_types/virologist.dm

```diff
@@ -11,6 +11,9 @@
 	wiki_page = "Infections" //WS Edit - Wikilinks/Warning
 	special_notice = "Don't release viruses, harmful or not, without the CMO's permission." //WS Edit - Wikilinks/Warning
 
+	skills = list(/datum/skill/healing = SKILL_EXP_NOVICE)
```

Comment:
This PR has so much stuff that's not even really that related to wounds holy shit
Also a bunch of one and two-liners that just change sharpness and similar

Reply:
I really need to gut out the skills thing that was added
Seems like the last ws coders thought making wound sounds only heard from people with novice in medical skills was a good idea

Reply:
job skill system was removed